package.json

```diff
@@ -2,8 +2,8 @@
   "private": true,
   "name": "@okta/okta-react-native",
   "title": "React Native Okta Sdk Bridge",
-  "version": "1.9.2",
-  "types": "types/index.d.ts",
+  "version": "1.9.3",
+  "types": "dist/types/index.d.ts",
```

Comment:
As I understand it, the package for npm is built in `dist/`, and thus will have changes to the code.  (For example, notice that `private: true` above, but in the package.json that gets put in `dist/`, `private` will not be true)

Since the package.json that is taken from `dist/` will be corrected, what you want here is to have the package.json that works with our files pre-bundling, for local development.

Reply:
See also this comment: https://github.com/okta/okta-react-native/pull/84#issuecomment-789880639

Reply:
I wonder if it's commong in JS world to have custom build script?